--- mozjs_icu_collator-153.3.0/Cargo.toml.published
+++ vendor/mozjs-icu-collator/Cargo.toml
@@ -106,7 +106,7 @@
 package = "mozjs_icu_collections"
 
 [dependencies.icu_locale]
-version = "~2.1.1"
+version = "^2.1.1"
 optional = true
 default-features = false
 
@@ -121,7 +121,7 @@
 package = "mozjs_icu_normalizer"
 
 [dependencies.icu_properties]
-version = "~2.1.1"
+version = "^2.1.1"
 default-features = false
 
 [dependencies.icu_provider]
